pkg/republisher/uuid_republisher.go

```diff
@@ -31,11 +31,9 @@ func NewNotifyingUUIDRepublisher(uuidCollectionRepublisher UUIDCollectionRepubli
 func (r *NotifyingUUIDRepublisher) Republish(uuid, tidPrefix string, republishScope string) (msgs []*OKMsg, errs []error) {
 	isFoundInAnyCollection := false
 	isScopedInAnyCollection := false
+	foundInUC := false
 
-	for _, collection := range r.collections {
-		if republishScope != ScopeBoth && republishScope != collection.scope {
-			continue
-		}
+	auxRepublish := func(collection CollectionMetadata) []*OKMsg {
```

Comment:
What `auxRepublish` stands for as a name?

Reply:
yep, not a great name but didn't come with any better one. 
This is a closure function that defines a functionality that's is repeated many times in Republish but is not used anywhere outside, that's the reason not to be a normal function that's being called in Republish. Besides there is no need to deal with variable imports since those are within its scope. In few words it avoids having duplicate code and do not expose functionality externally.

Reply:
Not commenting on the inline function, just asking for the name - auxRepulish does not mean anything to me. Maybe I do not understand the meaning behind it, so this is why I am asking for what is its meaning.
I completely agree that there should be an function for that and no need for it to be imported from outside as is used only locally.

Reply:
Fair :), actually i'm open to name proposals

Reply:
I've renamed the function to `republishFrom := func(collection ..)` as part of the changes I propose for simplifying the republishing logic overall. With this change I consider this comment resolved.